easybuild/easyconfigs/v/VEP/VEP-96.0-foss-2019a-Perl-5.28.1.eb

```diff
@@ -0,0 +1,43 @@
+name = 'VEP'
+version = '96.0'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://www.ensembl.org/info/docs/tools/vep'
+description = """Variant Effect Predictor (VEP) determines the effect of your variants (SNPs, insertions, deletions,
+ CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions.
+Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently used routines in VEP."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://github.com/Ensembl/ensembl-vep/archive/release/']
+sources = ['%(version)s.tar.gz']
+checksums = ['338f4774683da45e53bdf9a2cfffad4803df53fc4d3ee72817a9e79e63e513f7']
+
+dependencies = [
+    ('Perl', '5.28.1'),
+    ('DBD-mysql', '4.050', versionsuffix),
+    ('BioPerl', '1.7.2', versionsuffix),
+    ('Bio-DB-HTS', '3.01', versionsuffix),
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
+
+exts_list = [
+    ('Bio::EnsEMBL::XS', '2.3.2', {
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/Ensembl/ensembl-xs/archive'],
+        'checksums': ['aafc59568cd1042259196575e99cdfeef9c0fb7966e5f915cfaf38c70885ffa5'],
+    }),
+]
+
+# Add sanity check of EnsEMBL-XS, vep easyblock already takes care of VEP
+sanity_check_paths = {
```

Comment:
@lexming if `sanity_check_paths` is included in an easyconfig file, the paths that would be checked by the easyblock are ignored, so you should list the `VEP`-related paths here too (or enhance the `VEP` easyblock to also check for the `EnsEMBL-XS` path if it's supposed to be there).

Reply:
Thanks for the clarification. Actually, `EnsEMBL-XS` is totally optional, `VEP` runs just fine without it. We've been using this build for more than a month without issues, but it might be better to have two builds. One with a regular `VEP` installation as in previous versions, and another one including `EnsEMBL-XS`.

Reply:
I'm not sure if it's worth creating two `VEP` easyconfigs...

What does having `EnsEMBL-XS` also installed imply w.r.t. `VEP`?
Can it just do more, or does it change some functionality to work different/better? Can you opt-in (or opt-out) of using `VEP`+`EnsEMBL-XS` when both are installed?

Reply:
It changes nothing on the user side, `EnsEMBL-XS` provides drop-in replacements in C of some `VEP` functions. But there is no opt-out, unless the Perl module of `EnsEMBL-XS` is unloaded. As I said, we are running a single build of `VEP/96.0` and it includes `EnsEMBL-XS`. No issues so far. So I might be being over-cautious by creating two easyconfigs.
I'll be out of the city for some days, so please feel free to make any changes to this PR.

Reply:
Moved `EnsEMBL-XS` to the default easyconfig of `VEP` and the sanity checks for this extension can now be set by the `vep` easyblock (https://github.com/easybuilders/easybuild-easyblocks/pull/1828).